kingfisher_scrapy/spidermiddlewares.py

```diff
@@ -161,16 +161,19 @@ def process_spider_output(self, response, result, spider):
                 size = spider.sample
             else:
                 size = 100
-
-            package = self._get_package_metadata(spider, data['package'], 'releases', item['data_type'])
-            iterable = util.transcode(spider, ijson.items, data['data'], 'releases.item')
+            if spider.data_type == 'release_package':
+                package_type = 'releases'
+            else:
+                package_type = 'records'
+            package = self._get_package_metadata(spider, data['package'], package_type, item['data_type'])
```

Comment:
Newline to separate code into logic blocks.

```suggestion

            package = self._get_package_metadata(spider, data['package'], package_type, item['data_type'])
```